### PR TITLE
my contribution to 1 april

### PR DESCRIPTION
**Pull request by jughu**

### CODE REVIEW

**Comment by FlattestGuitar** [comment]
contribution*

smh fam

**Comment by taukausanake** [comment]
I'm trying to figure out the chemical reasoning to this even though it's supposed to be April 1st thing, but I'm more curious now as to why this hasn't existed before. I mean, chemistry

**Comment by Anticept** [comment]
So basically, the new meme will be "Someone's been mixing sodium and chlorine in chemistry!"?

**Comment by tristan1333** [comment]
yousaltybro????????????????

File: code/modules/reagents/chemistry/recipes/others.dm [hunk 1]
```diff
@@ -326,6 +326,14 @@
 	required_reagents = list("sodium" = 1, "chlorine" = 1, "water" = 1)
 	result_amount = 3
 	mix_message = "The solution crystallizes with a brief flare of light."
+	
+/datum/chemical_reaction/sodiumchloride/on_reaction(datum/reagents/holder)
+	var/turf/T = get_turf(holder.my_atom)
+	T.visible_message("<span class='warning'>The solution generates a salty vapor!</span>")
+	for(var/mob/living/carbon/C in range(T, 1))
```
Comment (Fox-McCloud) [inline code/modules/reagents/chemistry/recipes/others.dm]: This should probably cover the entire station...

Comment (Crazylemon64) [inline code/modules/reagents/chemistry/recipes/others.dm]: https://github.com/Crazylemon64/Paradise/commit/84b1def3f0e1fc41a89ec09a929045ca5712080b

Comment (SamHPurp) [inline code/modules/reagents/chemistry/recipes/others.dm]: Add it to the atmospherics event to come out of all the vents and make everyone cry ;)

**Comment by Fox-McCloud** [comment]
o7

**Comment by Anticept** [comment]
I actually wanted this merged... :/

**Comment by jughu** [comment]
Theres only dreams

**Comment by taukausanake** [comment]
So how do you actually get table salt if not by mixing these chemicals? How am I going to piss on the chef by saying his food "needs more salt"?

**Comment by Anticept** [comment]
The joke was that it would "create a salty vapour" when mixed.

**Comment by taukausanake** [comment]
@Anticept I mean that doesn't answer my question but I guess I can code dive later

**Comment by Anticept** [comment]
I misinterpreted your question.

Salt is in salt shakers.

**Comment by taukausanake** [comment]
@Anticept How do I put more salt into it?

**Comment by Anticept** [comment]
It should just be a container. Get salt from chemistry and pour it in.

**Pull request closed without merging**